public/components/data_connections/components/manage_datasource_table.tsx

```diff
@@ -0,0 +1,162 @@
+/*
+ * Copyright OpenSearch Contributors
+ * SPDX-License-Identifier: Apache-2.0
+ */
+
+import {
+  EuiFlexGroup,
+  EuiFlexItem,
+  EuiIcon,
+  EuiInMemoryTable,
+  EuiLink,
+  EuiPage,
+  EuiPageBody,
+  EuiPageContent,
+  EuiTableFieldDataColumnType,
+  EuiText,
+} from '@elastic/eui';
+import _ from 'lodash';
+import React, { useEffect, useState } from 'react';
+import { DataConnectionsHeader } from './datasources_header';
+import { HomeProps } from '../home';
+import { DataConnectionsDescription } from './manage_datasource_description';
+import sparkSvg from '../icons/spark_logo.svg';
```

Comment:
Is this reviewed, to use a `Spark TM logo` in plugin?

Reply:
Not sure, I just added it - I can remove it and only show opensearch logo for now, wdyt @ps48 ?

Reply:
Need to check with @brijos on this.

Reply:
Let's remove this and not block the PR for now.

Reply:
Done in: [3caa45d](https://github.com/opensearch-project/dashboards-observability/pull/967/commits/3caa45d14ddcb05bee80002c51f41ba752ca095c)